templates/registration/login.html

```diff
@@ -0,0 +1,16 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Login</h1>
+
+  {% if form.errors %}
+    <p style="color: red">Invalid credentials</p>
+  {% endif %}
+
+  <form method="post" action="{% url 'login' %}">
```

Comment:
The form action URL is set to `{% url 'login' %}`. In Django, the login URL is typically included via `django.contrib.auth.urls`, which might not directly match `'login'`. Ensure that the URL name `'login'` corresponds to the correct URL pattern in your project's URL configuration.